libs/platform/src/lib/form/form-generator/dynamic-form-generator-checkbox/dynamic-form-generator-checkbox.component.html

```diff
@@ -1,6 +1,7 @@
 <ng-container [formGroup]="form">
     <ng-container [formGroupName]="formGroupName">
         <fdp-checkbox-group
+            [id]="id"
```

Comment:
I think we want an ID for each checkbox not just the whole group. Same for radio

Reply:
That's a tough question. I think having ID on the main group should be enough

Reply:
Resolved?